part_2/second_split/functions_v2.R

```diff
@@ -12,19 +10,20 @@ library(reshape2)
 library(tseries)
 library(knitr)
 
-# Remove points with "-" in the cell
+# Exploratory analysis functions ---------------------------
+
+# Summary: Evaluate whether the string with "-".
```

Comment:
Do this in the SQL call and remove from the R script.

Reply:
Corrected.